packages/laconia-config/test/FloatConfigConverter.spec.js

```
@@ -0,0 +1,60 @@
+const FloatConfigConverter = require("../src/FloatConfigConverter");
+
+describe("FloatConfigConverter", () => {
+  describe("when there is one env var set", () => {
+    let configConverter;
+
+    it("converts a float value to a float", async () => {
+      configConverter = new FloatConfigConverter();
+      const instances = await configConverter.convertMultiple({
+        tax: "80.80"
+      });
+      expect(instances).toHaveProperty("tax", 80.8);
+    });
+
+    it("throws an error when value is not a float", async () => {
+      configConverter = new FloatConfigConverter();
+      await expect(() =>
+        configConverter.convertMultiple({
+          tax: "foo"
+        })
+      ).toThrow(`Passed config:float "tax" = "foo" is not a valid float.`);
+    });
+
+    // Thoughts on this?
+    it("throws an error when value is an empty string", async () => {
```

Should we throw an error if the value is an empty string?